docker/Dockerfile.dist.ubuntu16.04

```diff
@@ -0,0 +1,33 @@
+FROM ubuntu:16.04
+
+RUN apt-get update && apt-get -y install \
+      curl \
+      git \
+      libssl-dev \
+      libtool \
+      libunwind8 \
+      libunwind-setjmp0-dev \
+      python \
+      software-properties-common \
+      unzip \
+      wget \
+      zip
+
+RUN \
+  echo oracle-java8-installer shared/accepted-oracle-license-v1-1 select true | debconf-set-selections && \
+  add-apt-repository -y ppa:webupd8team/java && \
+  apt-get update && \
+  apt-get install -y oracle-java8-installer && \
+  rm -rf /var/lib/apt/lists/* && \
+  rm -rf /var/cache/oracle-jdk8-installer
+ENV JAVA_HOME /usr/lib/jvm/java-8-oracle
+
+ADD artifacts /heron
+
+WORKDIR /heron
+
+#run heron installers
+RUN /heron/heron-tools-install-*-ubuntu16.04.sh && \
+    /heron/heron-client-install-*-ubuntu16.04.sh && \
+    /heron/heron-tools-install-*-ubuntu16.04.sh
```

Comment:
It's copied from `Dockerfile.dist.ubuntu14.04`. I'll remove the last line and build again to see whether it works or not